### PR TITLE
All Script amounts multiplied by 1e18

### PR DESCRIPTION
**Pull request by bh2smith**

### CODE REVIEW

File: test/utilities.js [hunk 1]
```diff
@@ -91,9 +91,10 @@ const registerTokens = async function(token_artifact, contract, token_owner, num
  */
 const setupEnvironment = async function(token_artifact, contract, token_owner, accounts, numTokens) {
   const tokens = await registerTokens(token_artifact, contract, token_owner, numTokens)
+  const amount = "100000000000000000000"
```
Comment (fleupold) [inline test/utilities.js:94]: nit: could use `web3.utils.toWei` as well?

File: scripts/deposit.js [hunk 2]
```diff
@@ -24,12 +25,13 @@ module.exports = async (callback) => {
     }
   
     const token = await ERC20Mintable.at(token_address)
-    const depositor_balance = (await token.balanceOf.call(depositor)).toNumber()
-    if (depositor_balance < amount) {
-      callback("Error: Depositor has insufficient balance.")
+    const depositor_balance = (await token.balanceOf.call(depositor))
+    if (depositor_balance.lt(new web3.utils.BN(amount))) {
+      console.log(`Deposit request of ${amount} < `)
```
Comment (fleupold) [inline scripts/deposit.js]: Do you want to keep this?

**Review by fleupold — APPROVED** [review]
Need to change the placeOrder script as well?

File: test/utilities.js (hunk 1 above)
Comment (bh2smith) [reply test/utilities.js:94]: I tried using it here, but no web3 was available in this script... Didn't think it was worth the time to import it.  Do you?

File: scripts/deposit.js (hunk 2 above)
Comment (bh2smith) [reply scripts/deposit.js]: Sorry, I forgot to include some changes (vsCode doesn't save auto-magically).

File: test/snapp_base.js [hunk 3]
```diff
@@ -787,7 +787,7 @@ contract("SnappBase", async (accounts) => {
       )
     })
 
-    it("Generic claim", async () => {
+    it.only("Generic claim", async () => {
```
Comment (josojo) [inline test/snapp_base.js:790]: please take the only out

Comment (bh2smith) [reply test/snapp_base.js:790]: Oh yes of course sorry! I will push this as a "hot fix" since its already been merged!

**Comment by fleupold** [comment]
This is a typical error that can easily happen. 
Can we add a travis check that makes sure none of the test files have an .only in the when we commit?

**Pull request merged**